app/models/post.rb

```diff
@@ -10,7 +10,7 @@ class Post < ApplicationRecord
   after_save :update_counter
 
   def update_counter
-    author.update(posts_count: author.posts.size)
+    user.update(posts_count: author.posts.size)
   end
 
   def recent_comment
```

Comment:
- [ ] There is an error that is causing Posts to not be created. You are using user.update(posts_count: author.posts.size), but user is not defined in this context. Since the Post model has a belongs_to :author association with the User model, you should use author instead of user.

Here's the corrected update_counter method:


````
def update_counter
  author.update(posts_count: author.posts.size)
end
```